SocialFilter/sections/specific.txt

```diff
@@ -63,6 +63,9 @@ allets.ru,annahelp.ru,newslenta.com,kogdavyydet.com,u74.ru,wifika.ru,mater.com.u
 !
+corearoadbike.com##img[onclick*="twitter.com/share"]
+corearoadbike.com##img[onclick*="facebook.com/sharer/sharer.php"]
+corearoadbike.com##form[name="form_view"] ~ table[align="center"]
```

Comment:
The site does not open even with VPN. Could you provide some screenshots for those rules?

Reply:
With DevTool?

Reply:
Yes, it would be nice.

Reply:
<details><summary>Screenshots</summary>

![Screenshot3](https://user-images.githubusercontent.com/98787049/209802792-002e39d4-6b28-4534-9040-9f07e42dd4f9.png)
![Screenshot1](https://user-images.githubusercontent.com/98787049/209802802-8998d95f-98f5-4705-b419-790e3e103559.png)
![Screenshot2](https://user-images.githubusercontent.com/98787049/209802809-6a6b185b-bcd8-4a5d-a6ff-bde643670181.png)

</details>

Do you need more screenshot?

Reply:
They look ok. I don't need anymore screenshots. thanks!